includes/forms/form-customizer.php

```diff
@@ -68,15 +97,16 @@ function customize_controls_init() {
 		 * @date    27/05/2015
 		 * @since   ACF 5.2.3
 		 *
-		 * @param   $instance (array) widget settings
-		 * @param   $new_instance (array) widget settings
-		 * @param   $old_instance (array) widget settings
-		 * @param   $widget (object) widget info
-		 * @return  $instance
+		 * @param array  $instance (array) widget settings.
+		 * @param array  $new_instance (array) widget settings.
+		 * @param array  $old_instance (array) widget settings.
+		 * @param object $widget (object) widget info.
+		 * @return array                                $instance Widget settings.
```

Comment:
This spacing seems off to me.

```suggestion
		 *
		 * @return array $instance Widget settings.
```

Reply:
Fixed!